packages/fast-check/src/arbitrary/_internals/helpers/FloatHelpers.ts

```diff
@@ -17,12 +16,18 @@ const INDEX_POSITIVE_INFINITY = 2139095040; // floatToIndex(MAX_VALUE_32) + 1;
 /** @internal */
 const INDEX_NEGATIVE_INFINITY = -2139095041; // floatToIndex(-MAX_VALUE_32) - 1
 
+const buffer = new ArrayBuffer(4);
+const view = new DataView(buffer);
```

Comment:
**Perfect! It even works with node 8** which is the currently minimal supported version for fast-check (will be dropped soon, but so far v3 is supposed to support it).